### ISSUE TITLE
BUG: tuple default argument representation

### ISSUE THREAD

**Issue by larsoner** [sphinx-doc/sphinx#8052]
**Describe the bug**
Documenting a function with `def func(x=(1, 2))` shows up as `def func(x=1, 2)` when processed by Sphinx.

According to a `git bisect` this regression was introduced by c4d7f4d6c8f02dde6fb68ff4013caee2baee0abc

**To Reproduce**
From https://github.com/numpy/numpydoc/pull/291, steps to reproduce the behavior:
```
$ git clone https://github.com/larsoner/numpydoc
$ cd numpydoc
$ git checkout -b tuple origin/tuple
$ cd numpydoc/tests/tinybuild
$ make
<click on my_function in the built doc that pops up>
```

**Screenshots**

On commit 822625d14c0cad6607553079b31930ff9e2de717 (correct):

![Screenshot from 2020-08-04 11-56-23](https://user-images.githubusercontent.com/2365790/89316054-9476ed00-d649-11ea-9226-2e43506ecbb8.png)

On commit c4d7f4d6c8f02dde6fb68ff4013caee2baee0abc (incorrect):

![Screenshot from 2020-08-04 12-02-19](https://user-images.githubusercontent.com/2365790/89316625-5cbc7500-d64a-11ea-88b7-bb417ac1668b.png)

**Environment info**
- OS: Linunx
- Python version: 3.8
- Sphinx version: above
- Sphinx extensions: numpydoc, autodoc, autosummary

**Comment by hsfzxjy** [comment]
Hi. The bug seems still to appear in v3.2.0?

**Comment by larsoner** [comment]
Looks like this was fixed by #8265
